onnxtech.txt

```diff
@@ -3,165 +3,394 @@ apis: c
 apis: c#
 apis: c++
 apis: c++: hip
-apis: c++: hipify
 apis: csharp
 apis: java
-apis: java: react
-apis: objective c
+apis: javascript
+apis: javascript: react
+apis: objective-c
 apis: python
 architectures
-architectures: amd
-architectures: apple silicon
-architectures: arm
-architectures: arm: aarch64
-architectures: arm: neon
-architectures: arm32
-architectures: arm64
-architectures: avx
-architectures: avx512
 architectures: cpu
+architectures: cpu: arm
+architectures: cpu: arm: apple silicon
+architectures: cpu: arm: aarch64
+architectures: cpu: arm: neon
+architectures: cpu: x86
+architectures: cpu: x86: extensions
+architectures: cpu: x86: x64
+architectures: cpu: power
+architectures: accelerator
+architectures: accelerator: vpu
+architectures: accelerator: npu
```

Comment:
Regrouped these to have a richer hierarchy, based on what they are.
avx/avx512 (and mmxc, sse, sse2, sse3, fma, etc) are all x86 extensions, so would fold under that